web/app.js

```diff
@@ -2759,6 +2780,18 @@ function onKeyUp(evt) {
 function onKeyDown(evt) {
   this._isCtrlKeyDown = evt.key === "Control";
 
+  if (
+    this.editorUndoBar?.isOpen &&
+    evt.keyCode !== 9 &&
+    evt.keyCode !== 16 &&
+    (getActiveOrFocusedElement() !== this.editorUndoBar.undoButton ||
```

Comment:
Similar to other code in this file, can't we do the following here (and avoid having to introduce a getter just for this one spot)?
```suggestion
    (getActiveOrFocusedElement() !== this.appConfig.editorUndoBar.undoButton ||
```